lib/travis/api/v3/renderer/active.rb

```diff
@@ -1,3 +1,5 @@
+require 'travis/api/v3/renderer/collection_renderer'
```

Comment:
Both files are in the same directory and alphabetically the base classes come after this file name. I think collection_renderer and model_renderer are both only base classes, all the other base classes for our V3 structure (like: renderer, query, remote_query) are outside the sub directories. I think we should move these 2 base class files to the main V3 directory instead of inside the renderer subdirectory.